numba/core/compiler_lock.py

```diff
@@ -1,17 +1,21 @@
 import threading
 import functools
+import numba.core.event as ev
 
 
 # Lock for the preventing multiple compiler execution
 class _CompilerLock(object):
     def __init__(self):
         self._lock = threading.RLock()
+        self._duration = 0
```

Comment:
Ought this be a float?

Reply:
i'm just relying on Python duck typing it.

Reply:
actually, that's a leftover dead code i missed